server/routerlicious/packages/services/src/throttler.ts

```diff
@@ -51,13 +51,19 @@ export class Throttler implements IThrottler {
         // check cached throttle status, but allow operation through if status is not yet cached
         const cachedThrottlerResponse = this.throttlerResponseCache.get(id);
         if (cachedThrottlerResponse && cachedThrottlerResponse.throttleStatus) {
-            this.logger?.info(`Throttling ${id} for ${cachedThrottlerResponse.retryAfterInMs}ms: ${
-                cachedThrottlerResponse.throttleReason
-            }`);
+            const retryAfterInSeconds = Math.ceil(cachedThrottlerResponse.retryAfterInMs / 1000);
+            this.logger?.info(`Throttled: ${id}`, {
+                messageMetaData: {
+                    key: id,
+                    reason: cachedThrottlerResponse.throttleReason,
+                    retryAfterInSeconds,
```

Comment:
Should this be retryAfterInSeconds:  retryAfterInSeconds?

Reply:
Nope! That syntax actually throws an ESLint error I believe. I'm struggling to remember the name for this type of syntax, but basically this sets the key of the same name as the variable name to the value of the variable.
```js
const key = "value";
const obj1 = {
  key,
};
const obj2 = {
  key: key,
};
const obj3 = {
  key: "value",
};

assert(obj1.key === obj2.key === obj3.key)
```